### PR TITLE
add a comment about chvt keybindings

### PR DESCRIPTION
**Pull request by sevz17**
Closes: https://github.com/djpohly/dwl/issues/427

### CODE REVIEW

**Comment by sevz17** [comment]
Probably there is a better way to let the users know they are important, I am open to suggestions.

**Comment by pm4rcin** [comment]
I think it's fine to merge because IMO people who use standalone WMs know what a VT is. It's not like many new to linux users start with WMs rather than DE. I wonder if any linux user started with WMs.

File: config.def.h [hunk 1]
```diff
@@ -146,6 +146,9 @@ static const Key keys[] = {
 
 	/* Ctrl-Alt-Backspace and Ctrl-Alt-Fx used to be handled by X server */
 	{ WLR_MODIFIER_CTRL|WLR_MODIFIER_ALT,XKB_KEY_Terminate_Server, quit, {0} },
+	/* Ctrl-Alt-Fx is used to switch to another VT, if you don't know what a VT is
+	 * Do not remove them
```
Comment (pm4rcin) [inline config.def.h:150]: Not sure if capitalization of "Do" is needed. Also I don't get what does the * in that line do? I guess it's some convention or smth.

Comment (sevz17) [reply config.def.h]: Ups, it was a typo

**Pull request merged**